File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int left , right;
        left = nums[0];
        for (int i=1; i<nums.size(); i++) {
            right = nums[i];
            if (left == right)
                return true;
            else;
                left = nums[i];
        }
        return false;
    }
};

File: 217-contains-duplicate/contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        freq = {}
        for i in nums:
            freq[i] = freq.get(i, 0) + 1
        for x in freq:
            if freq[x] > 1:
                return True
        return False

        